cluster/ci/config/prow/config.yaml

```diff
@@ -3939,6 +3943,168 @@ postsubmits:
         - --target=[images]
         - --promote
 
+  openshift/prometheus-operator:
+  - name: branch-ci-openshift-prometheus-operator-images
+    agent: kubernetes
+    labels:
+      artifacts: images
+    branches:
+    - openshift-master
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      containers:
+      - name: test
+        image: ci-operator:latest
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-openshift-prometheus-operator
+              key: master.json
+        command:
+        - ci-operator
+        args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --target=[images]
+        - --promote
+
+  openshift/prometheus-config-reloader:
```

Comment:
This repo doesn't exist, so you need to fold this into openshift/prometheus-operator (which ci-operator does for you automatically)